### PR TITLE
Add ability to scroll context element horizontally

### LINKED ISSUE #140: The framework hasn't step for horizontal page scrolling
Problem: the framework hasn't step for horizontal page scrolling.

Our application requires horizontal scrolling of the page, so as the user scrolls the page horizontally, the data continues to be loaded. Accordingly, to make sure that all the data has arrived and there are no differences between the attributes (columns) in the database and the application, you need to scroll horizontally on the page, only after that you can clearly judge that all the data has arrived and the page is loaded without visible modal windows with errors.
In our application, the scroll looks like this:
2019-11-01 14_43_26-Window

Could you please add this step in Framework,
Kindly regards

### LINKED ISSUE #260: Could you please add an integration test for element scroll
Could you please add an integration test

_Originally posted by @ikalinin1 in https://github.com/vividus-framework/vividus/pull/259/files_

### PR DESCRIPTION
**Pull request by ikalinin1**
Closes #140
Closes #260

### CODE REVIEW

File: vividus-plugin-web-app/src/main/java/org/vividus/ui/web/action/IJavascriptActions.java [hunk 1]
```diff
@@ -63,6 +63,10 @@
 
     void scrollToStartOf(WebElement element);
 
+    void scrollHorizontallyToStartOf(WebElement element);
```
Comment (valfirst) [inline vividus-plugin-web-app/src/main/java/org/vividus/ui/web/action/IJavascriptActions.java]: `scrollToLeftOf` ? to keep the same naming scheme

File: vividus-plugin-web-app/src/main/java/org/vividus/ui/web/action/IJavascriptActions.java [hunk 2]
```diff
@@ -63,6 +63,10 @@
...
+
+    void scrollHorizontallyToEndOf(WebElement element);
```
Comment (valfirst) [inline vividus-plugin-web-app/src/main/java/org/vividus/ui/web/action/IJavascriptActions.java]: `scrollToRightOf`

File: vividus-plugin-web-app/src/main/resources/steps/defaults/scroll.steps [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Composite: When I scroll to the end of the page
+When I scroll context to BOTTOM edge
```
Comment (valfirst) [inline vividus-plugin-web-app/src/main/resources/steps/defaults/scroll.steps]: risky change, there could be cases, when some context is set and user wants to scroll the page, but not context

File: vividus-plugin-web-app/src/main/resources/steps/defaults/scroll.steps [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+Composite: When I scroll to the start of the page
+When I scroll context to TOP edge
```
Comment (valfirst) [inline vividus-plugin-web-app/src/main/resources/steps/defaults/scroll.steps]: risky change, there could be cases, when some context is set and user wants to scroll the page, but not context

File: vividus-plugin-web-app/src/main/resources/steps/defaults/scroll.steps (hunk 3 above)
Comment (ikalinin1) [reply vividus-plugin-web-app/src/main/resources/steps/defaults/scroll.steps]: It is, but shouldn't we cut it as soon as possible?

Comment (valfirst) [reply vividus-plugin-web-app/src/main/resources/steps/defaults/scroll.steps]: breaking changes are always painful for users, so I recommend to Deprecate these steps

**Pull request merged**